spec/lib/tasks/wp_cache_rake_spec.rb

```diff
@@ -0,0 +1,18 @@
+# spec/lib/tasks/wp_cache_rake_spec.rb
+require 'spec_helper'
+
+describe 'wp_cache' do
+  include_context 'rake'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.